README.md

```diff
@@ -1 +1,5 @@
-# ultrastrucural_mitochonria_morphology_analysis
\ No newline at end of file
+# Ultrastrucural mitochonria morphology analysis
+
+This repository contains the code used to analyze the ultrastructural morphology of mitochondria in electron microscopy images from [Suga et al. 2023](https://doi.org/10.1371/journal.pbio.3002246).
+The main goal of this project is to analyze the morphology of mitochondria in shRNA knockdowns of control and OPA1 cells.
```

Comment:
```suggestion
The main goal of this project is to demonstrate proof-of-concept for extracting high-content morphology measurements from electron microscopy (EM) images of mitochondria.

We will analyze the morphology of mitochondria tubular cristae and laminellar cristae in shRNA knockdowns of control and OPA1 cells.
```

Reply:
Please edit my suggestion as well, it would help to be a bit more specific in this repo's justification for existence :)